virtualization.go

```diff
@@ -294,3 +348,23 @@ func (v *VirtualMachine) Stop(fn func(error)) {
 func (v *VirtualMachine) StartGraphicApplication(width, height float64) {
 	C.startVirtualMachineWindow(v.Ptr(), C.double(width), C.double(height))
 }
+
+func macOSMajorVersion() int {
+	osver, err := syscall.Sysctl("kern.osproductversion")
+	if err != nil {
+		return 0
+	}
+	osverArray := strings.Split(osver, ".")
+	if len(osverArray) < 1 {
+		return 0
+	}
```

Comment:
Please remove this handling because the result will be more than or equal 1
https://go.dev/play/p/BJBkxhG0-lQ